File: examples-sdk/python/505-villog-tracing-bridge.py
#!/usr/bin/env python3
"""505-villog-tracing-bridge — Python SDK equivalent
Compile: vil compile --from python --input 505-villog-tracing-bridge.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/505-villog-tracing-bridge.ts
#!/usr/bin/env tsx
// 505-villog-tracing-bridge — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 505-villog-tracing-bridge.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
